WordPress/gutenberg
# Spacer: Convert knobs to controls in Storybook

## What?

Removes knobs in the `Spacer` story.

## Why?

Part of #35665

## Testing Instructions

1. `npm run storybook:dev`
2. See the `Spacer` story. The knobs should all be controls now, with the same default values.
